aspnetcore/host-and-deploy/blazor/client-side.md

```diff
@@ -184,6 +184,8 @@ For information on deploying to Azure App Service, see <xref:tutorials/publish-t
 
 A *standalone deployment* serves the client-side Blazor app as a set of static files that are requested directly by clients. A web server isn't used to serve the Blazor app.
```

Comment:
> A web server isn't used to serve the Blazor app.

I know this isn't part of this PR, but this statement sounds wrong. You still need a web server to host your Blazor app files.

Reply:
Serverless static file deployment. I should say it's not strictly required. I'll fix it.

Reply:
🤔 

> Any static file server is able to serve the Blazor app.